lib/rules/font-family-no-missing-generic-family-keyword/index.js

```diff
@@ -48,6 +62,14 @@ function rule(actual) {
 				return;
 			}
 
+			if (
+				fontFamilies.some((node) =>
+					optionsMatches(options, 'ignoreFontFamilies', node.value.toLowerCase()),
```

Comment:
```suggestion
					optionsMatches(options, 'ignoreFontFamilies', node.value),
```

We'll want to avoid using `toLowerCase` here as the user can use the [`i` flag](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/ignoreCase) in their regex if needed. (This differs from line 20, where we use `toLowerCase` because we are comparing against a predefined list of reference keywords there rather than a user-defined regex.)